### DISCUSSION TITLE
[求助]有什么好用免费的梯子吗？

### DISCUSSION THREAD

**Discussion by su2569** [share121/inter-knot#1556] [其他]
我现在用的是  ‘ikuuu’/‘dev-sidecar’  +油猴脚本的组合，时不时在 我想发/正在发  评论/文章 时给我来个连接超时，麻烦死，就想换个梯子，把这两个替换掉，大家有什么推荐吗？

**Comment by SSTV-10068** [comment]
您好，您的委托可以挂在SSTV上的广告位上喔~
SSTV广告位价格实惠，广告醒目吸人，[欢迎大家来抢购！](https://github.com/share121/inter-knot/discussions/1065)

**Comment by DerekH-233** [comment]
我用的就是这个，够好的了

**Comment by themagicmoon** [comment]
免费的几乎没啥好用的，我之前用的免费的上一秒还可以用，下一秒就用不了了

**Comment by SSTV-10068** [comment]
emmm，class of windows。貌似可以白嫖

**Comment by Xmug** [comment]
免费的不稳定，建议用付费的

**Comment by Myitian** [comment]
我的方案挺麻烦的，但访问速度一般还行，而且免费

我一般是不定期从一些油管主的视频/直播，和一部分公开网络地址抓一批节点，放入节点池，然后进行批量测延迟和测速，把无法使用或速度过低的从节点池删除（一般剩下100~500个），然后挑快的几个用（可用时间长短不一，短的几天，长的几月），一旦这几个挂掉就会再对节点池测个速，再挑快的用（部分节点速度不稳定，所以节点池会反复筛），直到节点池内基本都用不了，再重新下载一批加入节点池，如此反复

**Comment by saiki816** [comment]
我用的是“性价比机场”，虽然收费，但一个月只收一元(不是推广)

**Comment by su2569** [comment]
> emmm，class of windows。貌似可以白嫖

我就是在class of windows上用的ikuuu
时不时来一个
![114514](https://f.cdn-static.cn/81470_17234302725072.png)

**Comment by blqdswkl** [comment]
dageyun.com

**Comment by Yang66g** [comment]
我用的edge插件ilink，不过质量很差，之前被谷歌查到把我账号封了，申诉了三次都不管用
